dags/fxa_amplitude_export.py

```diff
@@ -0,0 +1,46 @@
+import datetime
+
+from airflow import models
+from utils.gcp import bigquery_etl_query
+from airflow.operators.subdag_operator import SubDagOperator
+from utils.amplitude import export_to_amplitude
+
+default_args = {
+    'owner': 'frank@mozilla.com',
+    'start_date': datetime.datetime(2020, 5, 1),
+    'email': ['telemetry-alerts@mozilla.com', 'frank@mozilla.com'],
+    'email_on_failure': True,
+    'email_on_retry': True,
+    'depends_on_past': True,
+    'retries': 1,
+    'retry_delay': datetime.timedelta(minutes=20),
+}
+
+dag_name = 'fxa_amplitude_export'
+
+with models.DAG(
+        dag_name,
+        schedule_interval='0 10 * * *',
```

Comment:
This follows the existing fxa job for events, but I'd like some info on where the ten-hour wait period came from.

Reply:
The 10 hour wait period was a very conservative guess since I don't know anything about guarantees on latency for the Stackdriver to BigQuery integration. The existing FxA ETL exists primarily to serve KPI calculations, so all we cared about was having the data ready early enough to have the KPI dashboard updated before 9:00 AM Pacific.

Running this at 1:00 UTC would _probably_ be fine but I have no hard data to back that up.

Reply:
I checked in with jbuck and got some info about the expected latency. I've recorded it here for future reference. 30 minutes should give us plenty of leeway.